evalml/data_checks/highly_null_data_check.py

```diff
@@ -12,17 +12,25 @@ class HighlyNullDataCheck(DataCheck):
     """Checks if there are any highly-null columns and rows in the input.
 
     Arguments:
-        pct_null_threshold(float): If the percentage of NaN values in an input feature exceeds this amount,
-            that column/row will be considered highly-null. Defaults to 0.95.
+        pct_null_col_threshold(float): If the percentage of NaN values in an input feature exceeds this amount,
+            that column will be considered highly-null. Defaults to 0.95.
+        pct_null_row_threshold(float): If the percentage of NaN values in an input feature exceeds this amount,
```

Comment:
Docstring nitpick: If the percentage of NaN values in an input row (not feature, which refers to col)